Gemfile

```diff
@@ -27,7 +27,7 @@ gem 'puma'
 gem 'rack-cors', :require => 'rack/cors'
 gem 'rack-jsonp'
 gem 'rakismet'
-gem 'sass-rails'
+gem 'sassc-rails', '~> 2.1'
```

Comment:
I personally prefer we leave this without a version number. We can pull in the latest with `bundle update`, and CI/manual testing will make sure nothing breaks.

Reply:
No problem.  I removed the version.